http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

```diff
@@ -88,13 +93,25 @@ object CommonTestData {
   val proxyUrlBase = proxyConfig.proxyUrlBase
   val monitorConfig = config.as[MonitorConfig]("monitor")
   val clusterBucketConfig = config.as[ClusterBucketConfig]("clusterBucket")
-  val vpcHelperConfig = VPCHelperConfig(
-    proxyConfig.projectVPCNetworkLabel,
-    proxyConfig.projectVPCSubnetLabel,
-    FirewallRuleName(proxyConfig.firewallRuleName),
-    proxyConfig.proxyProtocol,
-    proxyConfig.proxyPort,
-    List(NetworkTag(proxyConfig.networkTag))
+  val vpcInterpreterConfig = VPCInterpreterConfig(vpcConfig)
```

Comment:
are these needed or we can just use `Config.vpcInterpreterConfig`? `Config` is reading from configuration file anyway